src/app/pages/server-details/server-details.component.html

```diff
@@ -541,3 +541,21 @@ <h3 class="text-2xl" >Similar Servers </h3>
   class="absolute z-10 inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-secondary border border-solid border-emerald-400 rounded-lg">
   <div [innerHTML]="geekbenchHTML" ></div>
 </div>
+
+<div
+  id="toast-danger" #toastDanger
+  class="flex items-center w-full max-w-xs p-4 mb-4 text-white bg-secondary rounded-lg border boder-solid border-emerald-400" role="alert">
```

Comment:
IMO we should use the warning/error colors provided by Szucsi: maybe try the orange as the background and the red as border?